### PR TITLE
create QuizController

### PR DESCRIPTION
**Pull request by GeloPakDev**

### CODE REVIEW

**Review by kolesroma — APPROVED** [review]
Looks good for me

File: src/main/java/com/epam/emotionalhelp/model/Quiz.java [hunk 1]
```diff
@@ -23,7 +23,13 @@
 import java.time.ZonedDateTime;
 import java.util.Set;
 
-import static com.epam.emotionalhelp.model.util.ColumnName.*;
```
Comment (maximelyashevich) [inline src/main/java/com/epam/emotionalhelp/model/Quiz.java:26]: For static imports, you could use "import all", for non-static - no.

**Review by maximelyashevich — COMMENTED** [review]
Hi @GeloPakDev,
Please provide a quick code refactoring.

Regards,
Maks

File: src/main/java/com/epam/emotionalhelp/controller/config/CORSConfig.java [hunk 2]
```diff
@@ -1,8 +1,17 @@
...
+import org.springframework.web.servlet.config.annotation.EnableWebMvc;
+import org.springframework.web.servlet.config.annotation.WebMvcConfigurer;
+
+@Configuration
+@EnableWebMvc
+public class CORSConfig  implements WebMvcConfigurer {
+
+    @Override
+    public void addCorsMappings(CorsRegistry registry) {
+        registry.addMapping("/**").allowedOrigins("http://localhost:8080");
```
Comment (maximelyashevich) [inline src/main/java/com/epam/emotionalhelp/controller/config/CORSConfig.java:15]:
```suggestion
        registry.addMapping("/**");
```

**Review by maximelyashevich — COMMENTED** [review]
Hi @GeloPakDev,
Please apply my code suggestion and close this PR.

Regards,
Maks

**Review by maximelyashevich — APPROVED** [review]
Approved from my side.

**Review by kolesroma — APPROVED** [review]
Thanks for that. Approve

**Pull request closed without merging**